pandas/core/missing.py

```diff
@@ -324,7 +338,7 @@ def _interpolate_scipy_wrapper(
     elif method == "cubicspline":
         alt_methods["cubicspline"] = _cubicspline_interpolate
 
-    interp1d_methods = [
+    interp1d_methods: List[str] = [
```

Comment:
mypy cant figure this out on its own?

Reply:
I think maybe I needed this at some point but yes mypy gets this without the annotation with the file as is